lib/game.rb

```diff
@@ -73,8 +75,25 @@ def start
     end_game
   end
 
+  def print_welcome_to_battleship
```

Comment:
This is incredible work. I can see how hard you worked on this and I love that our game will feel like us!